packages/tests/server/regression/issue-2540-undefined-mutation.test.ts

```diff
@@ -215,12 +215,14 @@ describe('with superjson', () => {
       expect(await ctx.proxy.goodQuery.query()).toBe('good');
     });
     test('query, void response', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-confusing-void-expression
```

Comment:
I believe these are false negatives @JoshuaKGoldberg

Reply:
`.query()` returns `Promise<void>`. If we remove the comment then the rule complaints.

Reply:
`.query()` returns `Promise<void>`. If we remove the comment then the rule complains.

Also, `--report-unused-disable-directives` is enabled for `packages/tests`, so false negatives _should_ be blocked from existing.

Reply:
you're right - tricked by the `.toBe(undefined)` 😅